dev/MRTCore/mrt/Core/src/MRM.vcxproj

```diff
@@ -125,6 +125,8 @@
       <LinkIncremental>false</LinkIncremental>
       <GenerateDebugInformation>true</GenerateDebugInformation>
```

Comment:
no longer neede?

Reply:
IIUC, I think we still need this as it sets `/DEBUG`.

Oh, I think you're saying that the Release build doesn't need this since all configurations get it...?

Reply:
based on document, /Profile seems enough. not sure though

Reply:
IIUC, I think we need both /DEBUG and /PROFILE, as that causes /DEBUGTYPE:CV,FIXUP to be set.
(I suppose we could also set them manually as well if we wanted to be sure.)

Reply:
I really wish we have a simpler way to verify.

Reply:
Hm, re-reading the document: IIUC, setting `Profile=True` implies `EnableCOMDATFolding=No` and `OptimizeReferences=Yes`. 
But they recommend enabling `EnableCOMDATFolding`. But this can only be done by overriding the Profile settings.
So we'll need to add additional linker commands.

Reply:
For testing I think we can use maxdep with -l3 and see if it reports any errors.